demo_modules/slideshow/slideshow.js

```diff
@@ -318,6 +330,88 @@ class LoadYouTubePlaylistClientStrategy extends ClientLoadStrategy {
 }
 
 
+// LOAD VIDEO STRATEGY
+// This loading strategy knows how to load a normal HTML5-video.
+class LoadVideoServerStrategy extends ServerLoadStrategy {
+  constructor(config) {
+    super();
+    this.config = config;
+  }
+  loadMoreContent(opt_paginationToken) {
+    return Promise.resolve({
+      content: [this.config.file]
+    });
+  }
+  serializeForClient() {
+    return {video: this.config};
+  }
+}
+
+class LoadVideoClientStrategy extends ClientLoadStrategy {
+  constructor(config) {
+    super();
+    this.config = config;
+  }
+  init(surface) {
+    this.surface = surface;
+  }
+  loadContent(url) {
+    return new Promise((resolve, reject) => {
+      let video = document.createElement('video');
+      video.setAttribute('loop', 'loop');
+      video.setAttribute('width', this.surface.virtualRect.w);
+      video.setAttribute('height', this.surface.virtualRect.h);
+      
+      // PREPARE THE URL:
+      let extIndex = url.lastIndexOf('.');
+      let finalUrl = url;
+      // Remove extension.
+      if (extIndex != -1) {
+        finalUrl = url.substring(0, extIndex);
+      }
+      
+      // If we are talking about pre-split video, then we are reading files 
+      // with a specific pattern. Generate the appropriate name for this client.
+      if (this.config.presplit) {
```

Comment:
What's the plan for documenting this stuff? Specifically config options and how to split videos for this module.